src/components/table/table.js

```diff
@@ -236,6 +236,7 @@ class Table extends PureComponent {
       const header = columnLabel(columnText);
       return (
         <Truncate
+          key={Math.random()}
```

Comment:
Not sure if recommended way to deal with `the key` warnings ;)

Reply:
Looking at this code I would not tell it will raise those warnings and `key` should not be necessary here. That's interesting I would need to take a closer look.

Reply:
I think it's not, I probably left it by accident while trying to track down a warning, I'll check.

Reply:
the keys in `Table` weren't necessary, the warning coming from table has been fixed in `header-row-renderer-component`.